### PR TITLE
Bug 1224331 - Fix card previews when rotated/keyboard shown. r=etienne

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=627c4b1676e0d112861f6a09c3802623ed0ea4e5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2ad8a914f9205424a409a4f83da3e220a0f41deb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eb5ef7558be5d7c1562791198b15d827343da6f5

File: apps/system/js/app_window_manager.js [hunk 1]
```diff
@@ -728,6 +729,10 @@
       }
     },
 
+    _handle_cardviewprepare: function(evt) {
+      this.broadcastMessage('cardviewprepare');
+    },
```
Comment (etiennesegonzac) [inline apps/system/js/app_window_manager.js]: do we need this?
Looks like only the `InputWindowManager` needs the event and listens for it on the window (versus all appWindows listening on their element)

Comment (Cwiiis) [inline apps/system/js/app_window_manager.js]: Oh, in that case, no - cheers :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a0994dc06629795d4bf8a511f4d15a4677d092cd

**Pull request merged**